index.js

```diff
@@ -129,6 +132,29 @@ module.exports.makeApp = function () {
     next()
   })
 
+  /*
+   Route to check the status of the application by
+   calling the package_search of backend with row count as 0
+   to check whether the application is returning response or not
+ */
+  app.use('/status', async (req, res) => {
```

Comment:
@sagargg can we move this above auth routes as it doesn't require protection (authz)?